packages/demo/src/components/Login/index.js

```diff
@@ -1,35 +1,149 @@
+      let message = ''
+      if (hasValidationError) {
+        message = t.loginValidationEmailRequired
+      } else if (hasError) {
+        let translation = t.loginErrors[ $error.code ]
+        message = translation || $error.message
+      }
+      return getField({
+        field: 'user.signIn.$email',
+        icon: 'fa fa-user',
+        value: $email,
+        placeholder: t.loginEmailPlaceholder,
+        fieldType: 'email',
+        message: message
+      })
+    }
+
+    function getPassword () {
+      let hasValidationError = $validationsErrors && $validationsErrors.PASSWORD_EMPTY
+      let message = ''
+      if (hasValidationError) {
+        message = t.loginValidationPasswordRequired
+      }
+      return getField({
+        field: 'user.signIn.$password',
+        icon: 'fa fa-key',
+        value: $password,
+        placeholder: t.loginPasswordPlaceholder,
+        fieldType: 'password',
+        message: message
+      })
+    }
+
     return (
       <div className='modal is-active'>
         <div className='modal-background' />
         <div className='modal-content'>
           <div className='box'>
-            <h2 className='title'>{t.pleaseLogin}</h2>
-            <p className='control has-icon'>
-              <input className='input' type='text'
-                autoFocus
-                placeholder={t.loginUserPlaceholder}
-                value={''}
-                name='login'
-                />
-              <i className='fa fa-user' />
-            </p>
-            <p className='control has-icon'>
-              <input className='input' type='password'
-                placeholder={t.loginPasswordPlaceholder}
-                value={''}
-                name='password'
-                />
-              <i className='fa fa-key' />
-            </p>
+
+            <nav className='nav'>
+              <div className='nav-left'>
+                <h2 className='title'>{t.pleaseLogin}</h2>
+              </div>
+              <div className='nav-right'>
+                <div className='nav-item'>
+                  <LangSelector />
+                </div>
+              </div>
+            </nav>
+
+            {getEmail()}
+            {getPassword()}
+
+            <nav className='level'>
+              <div className='level-left' />
+              <div className='level-right'>
+                <div className='level-item'>
+                  <p className='control'>
+                    <button
+                      className='button is-primary'
+                      onClick={() => signInClicked()}
+                    >
+                      {t.loginSignInButton}
+                    </button>
+                  </p>
+                </div>
+                <div className='level-item'>
+                  <p className='control'>
+                    <button
+                      className='button is-primary'
+                      onClick={() => createUserClicked()}
+                    >
```

Comment:
Yes, please help me on this. I think to have signUp / signIn splited is very good :)